conf/xml/RobotStateProvider_iCub3.xml

```diff
@@ -3,7 +3,7 @@
 <robot name="iCub-Retargeting" build=0 portprefix="">
 
     <device type="iwear_remapper" name="XSenseIWearRemapper">
-        <param name="wearableDataPorts">(/XSensSuit/WearableData/data:o)</param>
+        <param name="wearableDataPorts">(/XSensSuit/WearableData/data:o /WearableData/HapticGlove/LeftHand/data:o)</param>
```

Comment:
I think this change is not supposed to be committed since you are committing a dedicated file for Xsens+gloves (`/RobotStateProvider_iCub3_Xsens_SenseGlove.xml`), and this file can the default for Xsens only demo.

Reply:
Actually, I think you can avoid to commit all the changes in this file (`RobotStateProvider_iCub3.xml`).